### PR TITLE
Make zombies drop tainted leather

### PR DESCRIPTION
**Pull request by Rail-Runner**
Make poisonous creatures drop poisonous leather/fur patches, not edible ones.
Also fixes name of tainted meat (so that it won't be "tainted meats" when you have more than one in inventory) and prevents zombified animals from dropping edible pelts.

### CODE REVIEW

**Comment by kevingranade** [comment]
Two problems with this.  From a balance point of view, zombie leather is
way too common. From a thematic point of view, zombie skin is going to be
too rotten/torn to use.

**Comment by Rivet-the-Zombie** [comment]
Yeah, normal zombies shouldn't be dropping usable hides. On the other hand, this is an excellent replacement for the normal raw leather that really thick-skinned zombie creatures currently drop.

Restrict it to zombie moose and such, and it would be good.

**Comment by chaosvolt** [comment]
Agreed, I'd suggest it stay just a replacement for tainted creatures that already drop raw hides/pelts.

Also of note, you can still eat cured hides. Should we split off cured hides into normal, cannibal, and toxic varieties, or is that a bit overkill?

**Comment by Rivet-the-Zombie** [comment]
>Also of note, you can still eat cured hides. Should we split off cured hides into normal, cannibal, and toxic varieties, or is that a bit overkill?

We could just make cured hides/pelts inedible and solve the whole issue.

**Comment by chaosvolt** [comment]
That would be saner, yes. ^^"

**Comment by Coolthulhu** [comment]
I'm a fair bit too late with this, but splitting off human leather into something would not be a bad idea.
After all, not everyone would wear human skin anything and even less people would drink water from water-human-skin.
In fact, most people wouldn't craft anything from human skin.

**Comment by Rivet-the-Zombie** [comment]
It would certainly be nice for axe-crazy cannibals to be able to make their own Ed Gein outfit - and even more fun if wearing it made you more frightening to NPCs - but that's outside the scope of this PR.

**Comment by Darkyhard** [comment]
> zombie leather is way too common

I think that most players won't even bother with tanning zombie leather, because zombies usually already drop leather items which can be cut for usable leather right away, and you can find more leather items at most places where there are a lot of zombies. And to use raw zombie leather, you need to tan it first, spending more time than it would take to find a leather item, and using up other items, at least salt, to prevent it from rotting away, and lye powder to make it actually usable.

>  zombie skin is going to be too rotten/torn to use.

True. But some large skin parts or zombies themself may not be so rotten or damaged.
I'm going to remove it from zombies whose leather is completely and obviously too rotten or damaged to be of any use.

> you can still eat cured hides.

As far as I know, they are inedible. Only raw pelts or hides are edible.

> After all, not everyone would wear human skin anything and even less people would drink water from water-human-skin.
In fact, most people wouldn't craft anything from human skin.

At first, most people wouldn't try to skin a human even if they were butchering a human corpse for food because they are starving and have nothing else to eat.
I can't think of a good way to handle this, except to make it possible to have certain traits/mutations as requirement for using certain components in recipes.

**Comment by Coolthulhu** [comment]
> I can't think of a good way to handle this, except to make it possible to have certain traits/mutations as requirement for using certain components in recipes.

Making human skins not spawn at all for non-cannibals, non-psychos etc. would be a good partial solution. It would result in minor oddities in shared worlds and players losing traits, but those are rare cases.
Though I'm not sure if human leather is a good idea for anything. Human skin is softer and thinner than cow or pig hide. Same goes for zombie skin, except for the armored mutants like hulks.

**Comment by Rivet-the-Zombie** [comment]
Still getting tainted hides from human zombies; please restrict them to zombified animals and then this will be ready for merge.

**Comment by Darkyhard** [comment]
Done.

File: data/json/monsters.json [hunk 1]
```diff
@@ -6267,7 +6267,7 @@
     "death_function":"NORMAL",
     "special_attacks":[["BRANDISH", 10]],
     "description":"Living in the woods, \nkilling for sport, \neating all the bodies, \nactual cannibal Shia LaBeouf.",
-    "flags":["SEES", "HEARS", "WARM", "BASHES", "POISON", "BLEED", "NO_BREATHE"],
+    "flags":["SEES", "HEARS", "WARM", "BASHES", "POISON", "LEATHER", "BONES", "BLEED", "NO_BREATHE"],
```
Comment (kevingranade) [inline data/json/monsters.json]: Shia is a human, sort of.

Comment (Rivet-the-Zombie) [inline data/json/monsters.json]: He only acts like he's from another planet.

Comment (Malkeus) [inline data/json/monsters.json]: Haha! Acts...

Comment (Darkyhard) [inline data/json/monsters.json]: Is this a reference to something?

Comment (Rivet-the-Zombie) [inline data/json/monsters.json]: A [silly internet meme](http://lmgtfy.com/?q=actual+cannibal+Shia+LaBeouf).

Comment (chaosvolt) [inline data/json/monsters.json]: [Derping Intensifies]

**Comment by Rivet-the-Zombie** [comment]
Please remove the LEATHER flag from Shia, and then this looks like it will be ready to merge.

**Comment by Darkyhard** [comment]
Done.

**Pull request merged**